substrate/frame/membership/src/lib.rs

```diff
@@ -500,8 +516,7 @@ mod benchmark {
 		}
 
 		clear_prime {
-			let m in 1 .. T::MaxMembers::get();
```

Comment:
I would not change this without reason since it will cause a breaking change in the `WeightInfo` trait and cause runtimes with it to not compile anymore.

Reply:
I understand that but it's redundant - there's no dependency on `m` related to the weight.
Seems wrong to leave it, especially since it's a minute change.

I will also update westend/rococo/example-node not to break anything.

If you strongly suggest to leave it as is, I'll do it like that.
But in that case I should also revert the rename of `reset_member` to keep it compatible.

Reply:
Thanks for the fast review btw 🙂